src/sdk.ts

```diff
@@ -937,6 +939,101 @@ export class OpenSeaSDK {
     }
   }
 
+  /**
+   * Transfer an asset. This asset can be an ERC20, ERC1155, or ERC721.
+   * @param options
+   * @param options.asset The Asset to transfer. tokenStandard must be set.
+   * @param options.amount Amount of asset to transfer. Not used for ERC721.
+   * @param options.fromAddress The address to transfer from
+   * @param options.toAddress The address to transfer to
+   * @param options.overrides Transaction overrides, ignored if not set.
+   */
+  public async transfer({
+    asset,
+    amount,
+    fromAddress,
+    toAddress,
+    overrides,
+  }: {
+    asset: AssetWithTokenStandard;
+    amount?: BigNumberish;
+    fromAddress: string;
+    toAddress: string;
+    overrides?: Overrides;
+  }): Promise<void> {
+    await this._requireAccountIsAvailable(fromAddress);
+    overrides = { ...overrides, from: fromAddress };
+    let transaction: Promise<ContractTransactionResponse>;
+
+    switch (asset.tokenStandard) {
+      case TokenStandard.ERC20: {
+        if (!amount) {
+          throw new Error("Missing ERC20 amount for transfer");
+        }
+        const contract = ERC20__factory.connect(
+          asset.tokenAddress,
+          this._signerOrProvider,
+        );
+        transaction = contract.transfer(toAddress, amount, overrides);
+        break;
+      }
+      case TokenStandard.ERC1155: {
+        if (asset.tokenId === undefined || asset.tokenId === null) {
+          throw new Error("Missing ERC1155 tokenId for transfer");
+        }
+        if (!amount) {
```

Comment:
This will throw on 0 amount. That's correct right?

Reply:
yes, should be good to throw early on this as the contract should revert with 0 amount